crates/opencascade-sys/src/lib.rs

```diff
@@ -292,6 +294,9 @@ pub mod ffi {
 
         #[rust_name = "set_mirror_axis"]
         pub fn SetMirror(self: Pin<&mut gp_Trsf>, axis: &gp_Ax1);
+        pub fn SetRotation (self: Pin<&mut gp_Trsf>, axis: &gp_Ax1, angle: f64);
```

Comment:
`cargo +nightly fmt` should fix up this line.

Again, I'll add automated checks and list steps to follow to run formatting soon.

Reply:
Fixed